roles/auth/defaults/main.yaml

```diff
@@ -12,3 +12,5 @@ dojot_auth_email_passwd: user_passwd
 
 dojot_kafka_port: 9092
 dojot_internal_api_config_port: 8001
+
+dojot_auth_protocol: https
```

Comment:
I think It should be something like
`dojot_gui_url_set_password: http://{{ dojot_domain_name }}/#/setPassword/`

And just pass dojot_gui_url_set_password in AUTH_RESET_PWD_VIEW

Or change for `dojot_protocol`;;;;